### PR TITLE
vsc network issue at vub

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: incidents/vub_20221118.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+meta_data:
+  title: VSC network unavailable
+  start_date: 2022-11-18 09:37:00
+  end_date:
+  affected: backbone, tier2_brussels
+  level: high
+  planned: no
+content: >
+
+  An upgrade of network equipement has caused the VSC network to be unavailable. The
```
Comment (wdpypere) [inline incidents/vub_20221118.yml:10]: typo in equipment.

File: incidents/vub_20221118.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+meta_data:
+  title: VSC network unavailable
+  start_date: 2022-11-18 09:37:00
+  end_date:
+  affected: backbone, tier2_brussels
+  level: high
+  planned: no
```
Comment (wdpypere) [inline incidents/vub_20221118.yml:7]: for some reason this goes up to the planned list when I look at https://status.vscentrum.be/

Comment (wpoely86) [reply incidents/vub_20221118.yml:7]: I know. See #17

Comment (wdpypere) [reply incidents/vub_20221118.yml:7]: if you look at https://status.vscentrum.be/backbone.html#incident_0003 it is planned, so there is a bug somewhere.

Comment (wdpypere) [reply incidents/vub_20221118.yml:7]: ah, issue already logged at https://github.com/vscentrum/status-page/issues/17

**Pull request merged**